gmso/core/potential.py

```diff
@@ -209,6 +209,39 @@ def __repr__(self):
         desc = "<Potential {}, id {}>".format(self._name, id(self))
         return desc
 
+    @classmethod
+    def from_template(cls, potential_template, parameters, topology=None):
+        """Create a potential object from the potential_template
+
+        Parameters
+        ----------
+        potential_template : gmso.lib.potential_templates.PotentialTemplate,
+                            The potential template object
+        parameters : dict,
+                    The parameters of the potential object to create
+        topology : gmso.Topology, default=None
+                   The topology to which the created potential object belongs to
+
+        Returns
+        -------
+        gmso.Potential,
```

Comment:
Should it be this?
```suggestion
        gmso.Potential :
```

Reply:
According to numpy docstring guide, there should neither be ',' nor ':'  in the docstring for Returns in this case:
```


Explanation of the returned values and their types. Similar to the Parameters section, except the name of each return value is optional. The type of each return value is always required:

Returns
-------
int
    Description of anonymous integer return value.

If both the name and type are specified, the Returns section takes the same form as the Parameters section:

Returns
-------
err_code : int
    Non-zero value indicates error code, or zero on success.
err_msg : str or None
    Human readable error message, or None on success.


```